src/server/middleware/csp.js

```diff
@@ -70,7 +70,7 @@ const csp = () => (req, res, next) => {
 
   res.scriptNonce = scriptNonce;
 
-  if (!process.env.ONE_DANGER_DISABLE_CSP) {
+  if (process.env.ONE_DANGER_DISABLE_CSP === 'false') {
```

Comment:
```suggestion
  if (process.env.ONE_DANGER_DISABLE_CSP !== 'true') {
```

This assertion is a bit safer. If somehow there was an invalid value, the header would still be included